Common/Data/Custom/SmartInsider/SmartInsiderTransaction.cs

```diff
@@ -0,0 +1,374 @@
+        /// </summary>
+        public string Currency { get; private set; }
+
+        /// <summary>
+        /// Denominated in Currency of Transaction
+        /// </summary>
+        public new decimal? Price { get; private set; }
+
+        /// <summary>
+        /// Number of shares traded
+        /// </summary>
+        public int? TransactionAmount { get; private set; }
+
+        /// <summary>
+        /// Currency conversion rates are updated daily and values are calculated at rate prevailing on the trade date
+        /// </summary>
+        public int? GBPValue { get; private set; }
+
+        /// <summary>
+        /// Currency conversion rates are updated daily and values are calculated at rate prevailing on the trade date
+        /// </summary>
+        public int? EURValue { get; private set; }
+
+        /// <summary>
+        /// Currency conversion rates are updated daily and values are calculated at rate prevailing on the trade date
+        /// </summary>
+        public int? USDValue { get; private set; }
+
+        /// <summary>
+        /// Free text which expains futher details about the trade
+        /// </summary>
+        public string NoteText { get; private set; }
+
+        /// <summary>
+        /// Percentage of value of the trade as part of the issuers total Market Cap
+        /// </summary>
+        public decimal? BuybackPercentage { get; private set; }
+
+        /// <summary>
+        /// Percentage of the volume traded on the day of the buyback.
+        /// </summary>
+        public decimal? VolumePercentage { get; private set; }
+
+        /// <summary>
+        /// Rate used to calculate 'Value (GBP)' from 'Price' multiplied by 'Amount'. Will be 1 where Currency is also 'GBP'
+        /// </summary>
+        public decimal? ConversionRate { get; private set; }
+
+        /// <summary>
+        /// Multiplier which can be applied to 'Amount' field to account for subsequent corporate action
+        /// </summary>
+        public decimal? AmountAdjustedFactor { get; private set; }
+
+        /// <summary>
+        /// Multiplier which can be applied to 'Price' and 'LastClose' fields to account for subsequent corporate actions
+        /// </summary>
+        public decimal? PriceAdjustedFactor { get; private set; }
+
+        /// <summary>
+        /// Post trade holding of the Treasury or Trust in the security traded
+        /// </summary>
+        public int? TreasuryHolding { get; private set; }
+
+        /// <summary>
+        /// Empty contsructor required for <see cref="Slice.Get{T}()"/>
+        /// </summary>
+        public SmartInsiderTransaction()
+        {
+        }
+
+        /// <summary>
+        /// Creates an instance of the object by taking a formatted CSV line
+        /// </summary>
+        /// <param name="line">Line of formatted CSV</param>
+        public SmartInsiderTransaction(string line) : base(line)
+        {
+            var csv = line.Split('\t');
```

Comment:
Another mismatch in nomenclature vs implementation. `csv` = comma separated values; `tsv` = tab separated values :)